src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -282,6 +309,33 @@ public sealed class AddPathCommandExtension : RunnerService, IActionCommandExten
 
         public void ProcessCommand(IExecutionContext context, string line, ActionCommand command, ContainerInfo container)
         {
+            var serverUrl = context.GetGitHubContext("server_url");
+            var isNotGithub = false;
+            if(!String.IsNullOrWhiteSpace(serverUrl) && serverUrl != "https://github.com")
+            {
+                isNotGithub = true;
+            }
+            
+            var allowUnsecureCommands = false;
+            bool.TryParse(Environment.GetEnvironmentVariable(Constants.Variables.Actions.AllowUnsupportedCommands), out allowUnsecureCommands);
```

Comment:
i would suggest checking both the environment variable as well as `env` from the workflow.

Folks who have many definitions may find it easier to set on the machine. Folks who don't have admin access to the machine find it easier to set on the workflow.